components/director/internal/domain/runtime/service.go

```diff
@@ -270,3 +282,24 @@ func (s *service) DeleteLabel(ctx context.Context, runtimeID string, key string)
 
 	return nil
 }
+
+func (s *service) ensureScenariosLabelDefinitionExists(ctx context.Context, tenant string) error {
+	ldExists, err := s.ldRepo.Exists(ctx, tenant, model.ScenariosKey)
```

Comment:
this is the same code as for application, we can create a service `ScenarioLabelService` and move this logic there. I don't see any problems with the situation when service depends on another service (even circular dependencies are solvable).